### PR TITLE
Problem: (fix #175) workflow to deploy by merging pr, dev and master …

### LINKED ISSUE #175: Problem: workflow to deploy by merging pr, dev and master is a bit complicated
Possible solution:
1. Deployment of staging: comment "/RUN" in PR only by limited collaborators i.e tomas and lesile, etc.
2. Deployment of production: only happen when merge to master

### PR DESCRIPTION
**Pull request by allthatjazzleo**
Solution:
	1. Deployment of staging: comment "/RUN" in PR only by limited collaborators i.e tomas and lesile, etc.
	2. Deployment of production: only happen when merge to master

### CODE REVIEW

**Review by landanhu — COMMENTED** [review]
let's try it out for staging tmr day time :)

**Comment by allthatjazzleo** [comment]
> let's try it out for staging tmr day time :)

sure but this commit has to be merged before try it out. or I can show it in my own repo?

**Review by calvinlauyh — APPROVED** [review]

File: .github/workflows/chain-docs.yml [hunk 1]
```diff
@@ -2,19 +2,44 @@ name: chain-docs
...
     - 'ci/**'
     - '!README.md'
+  issue_comment:
+    types: [created, edited]
 jobs:
 
   build:
     name: Build
     runs-on: ubuntu-latest
+    if: github.event_name == 'push' || (github.event.comment.body == '/RUN' && (github.actor == 'allthatjazzleo' || github.actor == 'tomtau' || github.actor == 'lezzokafka' || github.actor == 'yihuang' || github.actor == 'calvinlauco' || github.actor == 'foreseaz' || github.actor == 'CeruleanAtMonaco' || github.actor == 'devashishdxt' || github.actor == 'leejw51crypto'))
```
Comment (calvinlauyh) [inline .github/workflows/chain-docs.yml]: `/RUN` ... How about `/staging` haha?

```suggestion
    if: github.event_name == 'push' || (github.event.comment.body == '/staging' && (github.actor == 'allthatjazzleo' || github.actor == 'tomtau' || github.actor == 'lezzokafka' || github.actor == 'yihuang' || github.actor == 'calvinlauco' || github.actor == 'foreseaz' || github.actor == 'CeruleanAtMonaco' || github.actor == 'devashishdxt' || github.actor == 'leejw51crypto'))
```

**Review by lezzokafka — APPROVED** [review]

**Review by tomtau — APPROVED** [review]

**Pull request merged**